packages/components/src/local/form-elements/message-creator/types/props.d.ts

```diff
@@ -10,13 +10,18 @@ export default interface PropTypes {
   selectedForce: ForceData
   selectedRole: string
   selectedRoleName: string
```

Comment:
@lilitkarapetyan - could you include jsdoc comments for these new props, to explain what they do, please?

Reply:
Yes, I will include comments in that part of the code.